packages/chakra-ui/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@rjsf/chakra-ui",
+  "version": "3.2.1",
+  "description": "Chakra UI theme, fields, and widgets for react-jsonschema-form",
+  "main": "dist/index.js",
+  "module": "dist/chakra-ui.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "rimraf dist && tsdx build",
+    "test": "tsdx test",
+    "test:watch": "tsdx test --watch"
+  },
+  "engineStrict": false,
+  "engines": {
+    "node": ">=12"
+  },
+  "peerDependencies": {
+    "@chakra-ui/react": ">=1.7.3",
+    "@chakra-ui/icons": ">=1.1.1",
+    "framer-motion": ">=5.5.5",
+    "@rjsf/core": "^3.0.0",
+    "react": ">=16"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "Jonathan Blair <jonathanbblair@gmail.com>",
+  "contributors": [
+    "jzander <jeremy.zander@gmail.com>",
+    "Rodrigo Fuentes <rodrigofuentes@users.noreply.github.com>"
```

Comment:
@rodrigofuentes a tiny request. If you could kindly include "U.M Andrew <me@andrewmiracle.com>" I'll be glad

Reply:
> @rodrigofuentes a tiny request. If you could kindly include "U.M Andrew [me@andrewmiracle.com](mailto:me@andrewmiracle.com)" I'll be glad

Yeah, no problem.

Reply:
LGTM